storage/DevicegraphImpl.cc

```diff
@@ -559,6 +583,8 @@ namespace storage
 
 		if (is_disk(device))
 		    out << ", color=\"#ff0000\", fillcolor=\"#ffaaaa\"";
+		else if (is_md(device))
+		    out << ", color=\"#aaaa00\", fillcolor=\"#ffffaa\"";
```

Comment:
I think these colors should be defined outside. It doesn't belong into the code.